pkgs/applications/audio/dsf2flac/default.nix

```
@@ -0,0 +1,34 @@
+{ stdenv, fetchFromGitHub, autoreconfHook, boost, flac, id3lib, pkg-config
+, taglib, zlib }:
+
+stdenv.mkDerivation rec {
+  pname = "dsf2flac";
+  version = "0.1Rev54";
```

sorry, just noticed this. When not grabbing from an official release, you should use the format "unstable-YYYY-MM-DD". where the date is of the commit.
```suggestion
  version = "unstable-2018-01-01";
```